nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/frontend/package.json

```diff
@@ -18,28 +18,28 @@
   "dependencies": {
```

Comment:
Although there appear to be many changes in this file, most are a result of npm re-ordering the dependencies alphabetically after removing lodash-core and installing lodash@4.17.21 (this happens whenever an npm command is run that changes the package.json file).